modules/acquia_cms_common/acquia_cms_common.module

```diff
@@ -220,3 +221,17 @@ function _acquia_cms_common_update_view_display_options_style($view_name, $displ
     $view_storage->save($view);
   }
 }
+
+/**
+ * Implements hook_mail_alter().
+ */
+function acquia_cms_common_mail_alter(&$message) {
+  if (strpos($message['key'], 'template::') === 0) {
+    if (AcquiaDrupalEnvironmentDetector::isLocalEnv()) {
```

Comment:
@panshulK I think we can't use this, as this assumes everything as local if it's not an Acquia environment. So, it would restrict sending email, if acquia_cms is used in any other hosting platform.
@kdruckman

Reply:
Please also see if we can add some test coverage around this functionality.

Reply:
Sure @vishalkhode1 
Can we take the test creation part separately as this might need some more discussion?

Thanks